packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -239,16 +241,16 @@ class _CodeViewState extends State<CodeView>
 
     // Ensure the syntax highlighter has been initialized.
     // TODO(bkonyi): process source for highlighting on a separate thread.
-    if (parsedScript.script.source != null) {
-      if (parsedScript.script.source.length < 500000 &&
-          parsedScript.highlighter != null) {
-        final highlighted = parsedScript.highlighter.highlight(context);
+    if (parsedScript?.script.source != null) {
+      final scriptSource = parsedScript!.script.source!;
+      if (scriptSource.length < 500000) {
+        final highlighted = parsedScript!.highlighter.highlight(context);
 
         // Look for [TextSpan]s which only contain '\n' to manually break the
         // output from the syntax highlighter into individual lines.
         var currentLine = <TextSpan>[];
         highlighted.visitChildren((span) {
-          currentLine.add(span);
+          currentLine.add(span as TextSpan);
```

Comment:
The type of the parameter to `visitChildren`' is `InlineSpanVisitor`

Reply:
ok so type inference before null safety just wasn't warning us. Might add a TODO to change all these lists to <InlineSpan> instead of <TextSpan> if that makes sense.

Reply:
Added TODO